### PR TITLE
CB-18158 Event history listing not working for older DLs with datahub…

### PR DESCRIPTION
**Pull request by horadla23**
… CRN

See detailed description in the commit message.

### CODE REVIEW

File: datalake/src/main/java/com/sequenceiq/datalake/service/SdxEventsService.java [hunk 1]
```diff
@@ -131,7 +132,7 @@ private List<CDPStructuredEvent> retrievePagedCloudbreakServiceEvents(SdxCluster
             List<CloudbreakEventV4Response> cloudbreakEventV4Responses = ThreadBasedUserCrnProvider.doAsInternalActor(
                     regionAwareInternalCrnGeneratorFactory.iam().getInternalCrnForServiceAsString(),
                     () ->
-                            eventV4Endpoint.getPagedCloudbreakEventListByCrn(sdxCluster.getCrn(), page, size, false));
+                            eventV4Endpoint.getPagedCloudbreakEventListByCrn(getCloudbreakCrn(sdxCluster), page, size, false));
             return cloudbreakEventV4Responses.stream().map(entry -> convert(entry, sdxCluster.getCrn())).collect(toList());
```
Comment (nadamdb) [inline datalake/src/main/java/com/sequenceiq/datalake/service/SdxEventsService.java:136]: sanity check: in the response the DL crn should be used even if the stack crn is different, right?

Comment (horadla23) [reply datalake/src/main/java/com/sequenceiq/datalake/service/SdxEventsService.java:136]: i guess, since this is logic in dl service

**Review by nadamdb — APPROVED** [review]
LGTM, thanks!

**Comment by horadla23** [comment]
retest this please

**Review by oleewere — APPROVED** [review]

**Review by doktoric — APPROVED** [review]

**Pull request merged**